src/app/components/Curation/index.tsx

```diff
@@ -46,9 +48,8 @@ const Curation = ({
   headingLevel = 2,
   position = 0,
   curationLength = 0,
+  mostRead,
 }: CurationProps) => {
-  if (!promos.length) return null;
```

Comment:
Removed this line as MostRead will not contain a promos field and it therefore prevents it from rendering